marketplace-core/pom.xml

```diff
@@ -97,6 +97,10 @@
       <groupId>org.webjars.bower</groupId>
       <artifactId>angular-ui-bootstrap-bower</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Why do we need to introduce a dependency to kettle-core in marketplace-core?

marketplace-core should not depend on Kettle.
And we should only add a dependency to KEttle is we really really need it.